src/gui/sdl_mapper.cpp

```diff
@@ -68,9 +68,9 @@ enum BC_Types {
 	BC_Hold
 };
 
-#define BMOD_Mod1 0x0001
-#define BMOD_Mod2 0x0002
-#define BMOD_Mod3 0x0004
+#define BMOD_Mod1 MMOD1
+#define BMOD_Mod2 MMOD2
+#define BMOD_Mod3 MMOD3
```

Comment:
Instead of using MMOD1/2/3 directly, do these defines give us the option to swap them around on some hardware?

Reply:
I updated these defines because they have the same meaning and the same values as MMOD1, MMOD2, MMOD3. These are dosbox internal values.

And yeah - if it'll be needed we can swap the default scancode associated with Mod1 event and all the actions using it will be affected :) Or user can do this easily as well.